flink-staging/flink-scala-shell/start-script/start-scala-shell.sh

```diff
@@ -75,11 +75,20 @@ do
     fi
 done
 
+log_setting=""
+
+if [[ $1 = "yarn" ]]
+then
+FLINK_CLASSPATH=$FLINK_CLASSPATH:$HADOOP_CLASSPATH:$HADOOP_CONF_DIR:$YARN_CONF_DIR
+log=$FLINK_LOG_DIR/flink-$FLINK_IDENT_STRING-yarn-session-$HOSTNAME.log
```

Comment:
Why calling the log file like it is called when you start a dedicated yarn session via `yarn-session.sh`? Isn't this a bit confusing? Why not calling it `flink-$FLINK_IDENT_STRING-scala-shell-yarn-$HOSTNAME.log?